modules/altinnsalary/model/altinnreport.class.php

```diff
@@ -501,7 +524,88 @@ function generateArbeidsforholdArray($salary, $employee, $work_relation, $use_on
         self::checkIfEmpty($last_change_of_work_percentage, 'L&oslash;nnslipp: Mangler stillingsprosentendret p&aring; L' . $salary->JournalID, 'date');
       }
       $arbeidsforhold['sisteDatoForStillingsprosentendring'] = strftime('%F', strtotime($last_change_of_work_percentage));
+
+      # Check if ther is any furlough and stuff
+    }
+
+    // select all furlough active in the report period
+    $query_furlough = "SELECT * FROM workrelationfurlough WHERE WorkRelationID = " . $work_relation->WorkRelationID . " AND (('" . $this->period . "-01' BETWEEN FurloughStart AND FurloughStop) OR ((YEAR(FurloughStart) = YEAR('" . $this->period . "-01') AND MONTH(FurloughStart) = MONTH('" . $this->period . "-01')) OR (YEAR(FurloughStop) = YEAR('" . $this->period . "-01') AND MONTH(FurloughStop) = MONTH('" . $this->period . "-01'))))";
```

Comment:
Will between part of the query work for this example:
Sending for 2016-09.
Furlough is from 2016-05-23 and no end date.